### PR TITLE
Preliminary version of image resizing app

### PR DESCRIPTION
**Pull request by jason-alouda**
This is just a basic version. Please let me know of feedback for this template!

### CODE REVIEW

**Comment by tinafangkunding** [comment]
@yugasun can you help to review this pr? Thanks!

**Comment by yugasun** [comment]
@jason-alouda Why commit python dependencies? `PIL` and `Pillow` module contain so many files.

File: resizing/scf-resizing/Thumbnail.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+from qcloud_cos_v5 import CosS3Client
+from qcloud_cos_v5 import CosServiceError
+from qcloud_cos_v5 import CosClientError
+import sys
+import logging
+
+logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+
+appid = XXXXXX  # Please replace with your APPID. 请替换为您的 APPID
+secret_id = u'**************'  # Please replace with your SecretId. 请替换为您的 SecretId
```
Comment (yugasun) [inline resizing/scf-resizing/Thumbnail.py:18]: Maybe we can config SCF environment variables in `serverless.yml` and git it from`os.environ.get("secret_id")`.

Comment (jason-alouda) [reply resizing/scf-resizing/Thumbnail.py:18]: I think that's a great idea, will update it later

**Pull request closed without merging**